### PR TITLE
Add verbose option for logging

### PR DESCRIPTION
**Pull request by PleaseStopAsking**
This PR adds a new variable to enable verbose logging on the cron job for easier identification of issues as well as review of any purge/clean-up on snapshots.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -26,3 +26,4 @@ autorestic_info: false
 autorestic_check: false
 autorestic_cron: false
 autorestic_state: present
+autorestic_verbose: false
```
Comment (dbrennand) [inline defaults/main.yml]: Seeing as this is specifically related to the autorestic cron job, would we be able to make this variable name `autorestic_cron_verbose` ?

Also, then it pairs nicely with the `autorestic_cron` variable and we can have one after the other in the file. 🙂

**Review by dbrennand — CHANGES_REQUESTED** [review]
Hi @PleaseStopAsking 

Thank you for your contribution 🙂

Please add yourself to the contributors in the README.

Just one suggestion here, just a cosmetic change really to show the variable is related to `cron`.

File: defaults/main.yml (hunk 1 above)
Comment (PleaseStopAsking) [reply defaults/main.yml]: @dbrennand I was actually already thinking I should have used this naming scheme a few minutes after I submitted the PR but second guessed myself. I will make these changes shortly!

**Comment by dbrennand** [comment]
Hi @PleaseStopAsking - Don't worry about the lint errors, I will fix those - hopefully sometime tomorrow 🙂 Since it last ran Ansible-Lint has had loads of updates so new rules are being triggered and causing failures.

**Comment by PleaseStopAsking** [comment]
@dbrennand I went ahead and fixed the lint issues reported as they did not require any breaking changes. Feel free to run workflows again time permitting.

**Comment by dbrennand** [comment]
LGTM - Thank you @PleaseStopAsking for your contribution to this role 🙂

**Pull request merged**